clickhouse_driver/log.py

```diff
@@ -13,36 +37,26 @@
     'Notice',
     'Information',
     'Debug',
-    'Trace',
-    'Test',
+    'Trace'
 )
 
-num_priorities = len(log_priorities)
-
 
 def log_block(block):
     if block is None:
         return
-
     column_names = [x[0] for x in block.columns_with_types]
 
     for row in block.get_rows():
         row = dict(zip(column_names, row))
 
-        if 1 <= row['priority'] <= num_priorities:
-            priority = log_priorities[row['priority']]
+        if 1 <= row['priority'] <= 8:
+            row['priority'] = log_priorities[row['priority']]
         else:
-            priority = row[0]
+            row['priority'] = row[0]
 
         # thread_number in servers prior 20.x
-        thread_id = row.get('thread_id') or row['thread_number']
-
-        logger.info(
-            '[ %s ] [ %s ] {%s} <%s> %s: %s',
-            row['host_name'],
-            thread_id,
-            row['query_id'],
-            priority,
-            row['source'],
-            row['text']
-        )
+        row['thread_id'] = row.get('thread_id') or row['thread_number']
+
+        # put log block row into LogRecord extra
+        extra = {"server"+k: v for k, v in row.items()}
```

Comment:
Maybe this will be better?

```suggestion
        extra = {"clickhouse"+k: v for k, v in row.items()}
```

Also I don't see how does `_` added here, can you clarify?